data/campaigns/campaigns.ts

```diff
@@ -17,59 +17,81 @@ export const ALWAYS_SHOW = 0;
  */
 const CAMPAIGNS = {
```

Comment:
Is there a specific reason not to declare the schema upfront as a type and then declare this as a literal of that type, as opposed to doing generic gymnastics to infer the type? Do we need to support support different schemas for e.g. `locations`?

Reply:
```ts
type Campaigns = Record<AvalancheCenterID, Record<string, Campaign>>;

type Campaign = {
  enabled: boolean;
  locations: Record<string, Location>;
};

type Location = {
  frequency: number;
  startDate: Date;
  endDate: Date;
};

export const CAMPAGINS: Campaigns = {
  NWAC: {
    'q4-2023': {
      enabled: true,
      locations: {
        'observation-list-view': {
          frequency: ALWAYS_SHOW,
          startDate: new Date('2023-12-01'),
          endDate: new Date('2024-01-01'),
        },
        'map-view': {
          frequency: 16 * 60 * 60 * 1000, // 16 hours
          startDate: new Date('2023-12-22'),
          endDate: new Date('2024-01-01'),
        },
      },
    },
  },
};
```